src/components/Cart/CartProducts/CartProducts.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div v-for="item in cart"
+       :key="item.id"
+       class="np-cartProducts__item"
+       @mouseover="turnHoverOn(item)"
+       @mouseleave="turnHoverOff(item)">
+
+    <img :src="item.imgUrl"
+         alt="Food"
+         class="np-cartProducts__item__image"/>
+
+    <ProductsDetails v-if="!item.isHoveredOn" :item="item"/>
+    <ProductsHover v-else :item="item"/>
+  </div>
+
+  <div class="np-cartProducts__item__price">
+    <p>{{ $t('totalPrice') }}</p>
+    <p>${{ priceTotal.toFixed(2) }}</p>
+  </div>
+
+  <Buttons/>
+
+  <Modal v-if="showModal"/>
+</template>
+
+<script lang="ts">
+import {useStore} from "@/store";
+import {computed, defineComponent, ComputedRef} from "vue";
+import Buttons from "@/components/Cart/Steps/Buttons.vue";
+import {CartItem, CartState} from "@/store/interfaces";
+import ProductsDetails from "@/components/Cart/CartProducts/ProductsDetails.vue";
+import ProductsHover from "@/components/Cart/CartProducts/ProductsHover.vue";
+import Modal from "@/components/Cart/CartProducts/Modal.vue";
+
+export default defineComponent({
+
```

Comment:
redundant enter. this is simple object passed to defineComponent function. Don't use enter to separate attributes in object